sql/mob_groups.sql

```diff
@@ -8873,47 +8873,47 @@ INSERT INTO `mob_groups` VALUES (33,2237,135,'Kindred_Monk',600,0,1442,5500,0,77
 INSERT INTO `mob_groups` VALUES (47,1131,135,'Duke_Berith',0,32,714,12500,0,85,85,0);
-INSERT INTO `mob_groups` VALUES (48,3476,135,'Satellite_Longswords',600,0,0,4000,0,77,79,0);
+INSERT INTO `mob_groups` VALUES (48,3476,135,'Satellite_Longswords',600,0,3153,4000,0,77,79,0);
 INSERT INTO `mob_groups` VALUES (49,2578,135,'Marquis_Sabnak',0,32,1631,12500,0,85,85,0);
-INSERT INTO `mob_groups` VALUES (50,3467,135,'Satellite_Claymores',600,0,0,4000,0,77,79,0);
+INSERT INTO `mob_groups` VALUES (50,3467,135,'Satellite_Claymores',600,0,3154,4000,0,77,79,0);
 INSERT INTO `mob_groups` VALUES (51,819,135,'Count_Zaebos',0,32,521,12500,0,85,85,0);
-INSERT INTO `mob_groups` VALUES (52,3481,135,'Satellite_Tabars',600,0,2165,4000,0,77,79,0);
-INSERT INTO `mob_groups` VALUES (53,3469,135,'Satellite_Great_Axes',600,0,2164,4000,0,77,79,0);
+INSERT INTO `mob_groups` VALUES (52,3481,135,'Satellite_Tabars',600,0,3155,4000,0,77,79,0);
+INSERT INTO `mob_groups` VALUES (53,3469,135,'Satellite_Great_Axes',600,0,3156,4000,0,77,79,0);
 INSERT INTO `mob_groups` VALUES (54,2570,135,'Marquis_Andras',0,32,1624,12500,0,85,85,0);
 INSERT INTO `mob_groups` VALUES (55,134,135,'Andrass_Vouivre',0,128,0,7500,0,80,82,0);
 INSERT INTO `mob_groups` VALUES (56,2572,135,'Marquis_Cimeries',0,32,1625,12500,0,85,85,0);
 INSERT INTO `mob_groups` VALUES (57,1135,135,'Duke_Scox',0,32,717,12500,0,85,85,0);
-INSERT INTO `mob_groups` VALUES (58,3477,135,'Satellite_Scythes',600,0,0,4000,0,77,79,0);
-INSERT INTO `mob_groups` VALUES (59,3479,135,'Satellite_Spears',600,0,0,4000,0,77,79,0);
+INSERT INTO `mob_groups` VALUES (58,3477,135,'Satellite_Scythes',600,0,3157,4000,0,77,79,0);
+INSERT INTO `mob_groups` VALUES (59,3479,135,'Satellite_Spears',600,0,3158,4000,0,77,79,0);
 INSERT INTO `mob_groups` VALUES (60,2263,135,'King_Zagan',0,32,1452,12500,0,85,85,0);
 INSERT INTO `mob_groups` VALUES (61,4487,135,'Zagans_Wyvern',0,128,0,7500,2000,80,82,0);
 INSERT INTO `mob_groups` VALUES (62,818,135,'Count_Vine',0,32,520,12500,0,85,85,0);
-INSERT INTO `mob_groups` VALUES (63,3474,135,'Satellite_Kunai',600,0,0,4000,0,77,79,0);
-INSERT INTO `mob_groups` VALUES (64,3482,135,'Satellite_Tachi',600,0,0,4000,0,77,79,0);
+INSERT INTO `mob_groups` VALUES (63,3474,135,'Satellite_Kunai',600,0,3159,4000,0,77,79,0);
+INSERT INTO `mob_groups` VALUES (64,3482,135,'Satellite_Tachi',600,0,3160,4000,0,77,79,0);
 INSERT INTO `mob_groups` VALUES (65,2575,135,'Marquis_Gamygyn',0,32,1628,12500,0,85,85,0);
 INSERT INTO `mob_groups` VALUES (66,1707,135,'Goblin_Replica',0,128,3117,1000,1000,65,65,0);
 INSERT INTO `mob_groups` VALUES (67,2576,135,'Marquis_Nebiros',0,32,1629,12500,0,85,85,0);
 INSERT INTO `mob_groups` VALUES (68,2816,135,'Nebiross_Avatar',0,128,0,7500,0,85,85,0);
-INSERT INTO `mob_groups` VALUES (69,3472,135,'Satellite_Horns',600,0,0,4000,0,77,79,0);
-INSERT INTO `mob_groups` VALUES (70,4145,135,'Vanguard_Dragon',0,128,2559,30000,0,81,82,0);
```

Comment:
If Vanguard Dragons have been verified to be timed pops on retail, we need to set the respawn timer here as well.  Otherwise, they'll repop immediately after despawning.

Reply:
@EpicTaru, if I put 0 it will reappear directly after, no?

Reply:
I set respawntime to 300. I tested the vanguard dragon repop on my FFXI account and it repoped in 5 minutes